### ISSUE TITLE
Expand test coverage to failed ML convergence

### ISSUE THREAD

**Issue by Neves-P** [Neves-P/DAISIErobustness#41]
Once #40 is done, use the identified replicate causing the ML convergence failure to test flow of the pipeline when this happens.

**Comment by Neves-P** [comment]
This depends on #40 which is maked wontfix, hence I'm closing this issue
